admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,317 @@
+
+You can disable the detector by setting the `KUBE_ENABLE_NODE_PROBLEM_DETECTOR` environment variable  in the *_kubernetes/cluster/gce/config-default.sh_* file to false:
+
+[source,yaml]
+----
+KUBE_ENABLE_NODE_PROBLEM_DETECTOR=false
+----
+
+[[admin-guide-node-problem-detector-run]]
+== Running the Node Problem Detector in Other Environments
+
+To run the Node Problem Detector, in non-GCE environments, first create a xref:../dev_guide/daemonsets#dev-guide-daemonsets[daemonset] object.
+
+When xref:../dev_guide/daemonsets.adoc#dev-guide-daemonsets[creating daemonsets], the `nodeSelector` field is used to indicate the nodes on which the daemonset should deploy.
+
+. Copy the following code to a YAML file on your master:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-problem-detector-v0.1
+  namespace: kube-system
+  labels:
+    k8s-app: node-problem-detector
+    version: v0.1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: node-problem-detector
+        version: v0.1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      nodeSelector: <1>
+        type: infra
+      hostNetwork: true
+      containers:
+      - name: node-problem-detector
+        image: k8s.gcr.io/node-problem-detector:v0.1
+        securityContext:
+          privileged: true
+        resources:
+          limits: 
+            cpu: "200m"
+            memory: "100Mi"
+          requests:
+            cpu: "20m"
+            memory: "20Mi"
+        volumeMounts:
+        - name: log 
+          mountPath: /log 
+          readOnly: true
+      volumes:
+      - name: log
+        hostPath: <2>
+          path: /var/log/
+----
++
+<1> The node selector that determines on which nodes pod replicas should be deployed
+<2> Specify the location of your cluster log directory.
+
+. Edit the file to fit your environment. 
+
+. Run the following command to start node problem detector: 
++
+----
+oc create -f <your_yaml_file>.yaml
+----
+
+[[admin-guide-node-problem-detector-addon]]
+== Running the Node Problem Detector as an Addon Pod
+
+If you have your own cluster bootstrap solution and do not wat to overwrite the default configuration, 
+leverage the addon pod to further automate the deployment.
+
+Create the xref:admin-guide-node-problem-detector-run[`node-problem-detector.yaml`] file, 
+and save it to the addon pods directory *_/etc/kubernetes/addons/node-problem-detector_* on master node.
+
+[[admin-guide-node-problem-detector-overwrite]]
+== Overwrite the Configuration
+
+The default configuration is embedded when building the docker image of node problem detector.
+
+You can use a xref:../dev_guide/configmaps.adoc#dev-guide-configmaps[ConfigMap]
+to overwrite the configuration.
+
+. Change the configuraton files in config/.
+
```

Comment:
@joelsmith What is the `config/` directory? Do we use this in OpenShift or is it Kubernetes?